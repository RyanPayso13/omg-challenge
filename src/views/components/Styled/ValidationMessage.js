import styled from 'styled-components';

export const ValidationMessage = styled.p`
    color: red;
`;