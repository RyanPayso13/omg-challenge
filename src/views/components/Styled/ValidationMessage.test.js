import React from 'react';
import { render } from '@testing-library/react';
import { ValidationMessage } from './ValidationMessage';

describe('<ValidationMessage />', () => {

    it('should work', () => {
        const { container } = render(<ValidationMessage />);
        expect(container.firstChild).toMatchSnapshot();    
    });

});
