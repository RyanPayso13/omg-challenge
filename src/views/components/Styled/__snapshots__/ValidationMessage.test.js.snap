// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ValidationMessage /> should work 1`] = `
<p
  class="sc-bdVaJa eUkwbj"
/>
`;
